h2o-py/h2o/automl/_estimator.py

```diff
@@ -195,13 +196,8 @@ def __init__(self,
             More models can be trained on an existing AutoML project by specifying the same project name in multiple calls to the AutoML function
             (as long as the same training frame, or a sample, is used in subsequent runs).
         :param exclude_algos: List the algorithms to skip during the model-building phase. 
-            The full list of options is: 
-                ``"DRF"`` (Random Forest and Extremely-Randomized Trees),
-                ``"GLM"``,
-                ``"XGBoost"``,
-                ``"GBM"``,
-                ``"DeepLearning"``,
-                ``"StackedEnsemble"``. 
+            The full list of options is: ``"DRF"`` (Random Forest and Extremely-Randomized Trees), ``"GLM"``, ``"XGBoost"``, ``"GBM"``,
+            ``"DeepLearning"``, ``"StackedEnsemble"``.
```

Comment:
I don't know how you feel about this, but personally each time there are let's say >3 options like this, I think it's much nicer for the user to have them rendered as some kind of list in the generated doc. Also it allows us to put a description or quick explanation next to some items, without being visually annoying.
I tried the following approaches:
1. proper list:
```suggestion
            The full list of options is:
            
                * ``"DRF"`` (Random Forest and Extremely-Randomized Trees)
                * ``"GLM"``
                * ``"XGBoost"``
                * ``"GBM"``
                * ``"DeepLearning"``
                * ``"StackedEnsemble"`` 
                
```
2. code block: 
```suggestion
            The full list of options is::
            
                "DRF" (Random Forest and Extremely-Randomized Trees)
                "GLM"
                "XGBoost"
                "GBM"
                "DeepLearning"
                "StackedEnsemble" 
                
```
My preference goes for the second one: the rendering is more condensed but visually very easy to identify all the options.  Also note that in this case, there's no need for backsticks to highlight the options. wdyt?

Not asking you to change all the long lists here, but this one could be a good start and serve as an example for further improvements.